doc/reference/algorithms/bipartite.rst

```diff
@@ -27,6 +39,7 @@ Matching
    eppstein_matching
    hopcroft_karp_matching
    to_vertex_cover
+   maximum_matching
```

Comment:
Since this one is a true alias (`maximum_matching = hopcroft_karp_matching`), should we consider an approach similar to #4074, i.e. give `maximum_matching` it's own docstring that simply references `hopcroft_karp` (or vice-versa). It's probably not a big deal, but it would prevent the same docstring being parsed/evaluated twice during the doc build process.

Reply:
I'm fine with a fancier way of making this work. I think we do this in a couple of places -- have a function name defined by "monkey-patching" rather than setting up a function which calls another. If other packages have a better way of doing that, it'd be nice.  (I recall that ```out_edges``` and ```edges``` are methods, but they are just pointers to the same function.)

Reply:
I just handled this the way we did with #4074 which seems like a good way to do it.